src/ChannelServer/Skills/Skill.cs

```diff
@@ -171,7 +171,7 @@ public void IncreaseOverheat()
 			Send.ZC_OVERHEAT_CHANGED(this.Character, this);
 
 			// TODO: Apply overheat cooldown
-			if (this.IsOverheated)
+			if (this.IsOverheated || !this.CanOverheat)
```

Comment:
Did you add this because the packets actually need to be sent, or because officials send them, even if the skill can't overheat?

Reply:
So if you don't add that, the non-overheat skills don't go on cooldown. You didn't notice that? The official server still sends both packets even if a server can't overheat.

Reply:
Oh, makes sense. I haven't looked too much into skills yet. A comment would help here^^

Reply:
Removed, not a part of this PR.